### ISSUE TITLE
Corporate console throws error 'Request Failed' if while CCLA, the github user(who never had LFID) identifies himself as CLA Manager and completes the remaining steps

### ISSUE THREAD

**Issue by ashav-github** [communitybridge/easycla#2293]
Steps:

1. Have a github user who creates PR and gated. This github user never had LFID.
2. While signing CCLA, add a new org and select himself as CLA Manager
3. It redirects you to login page where you create a new account 
**Note: The contributor having github username: 'kapilk1435' while creating new LF account added a different email: ashav+kapilk1435@proximabiz.com which is different than his github email id: kapilk@proximabiz.com**
4. Verify email. 
5. After email verification - it loses the track where to go and page says 'The page you are looking for was not found' - This issue is already reported #2288
5. However, if I login to corporate console, login is successful but it throws error

Actual Result: Failed to get user by email: kapilk@proximabiz.com, error: user not found
![Screenshot from 2020-12-09 20-26-12.png](https://images.zenhubusercontent.com/5f6b335ce9044e3973ea11f9/6c0bea60-0f35-451d-bcc5-d150d360bbd1)

Expected Result: Since the contributor has created LF account with different email so after login to copr console it should not search for the email associated with Github, it should pick the email created during LF account.

**Comment by makkalot** [comment]
Blocked till the company creation flow is resolved again.

**Comment by makkalot** [comment]
I tested this today again : 

When I did `However, if I login to corporate console, login is successful but it throws error` : 
it was able to log in properly with the newly created lfid not with the github email. I think this seems to be resolved.
![Screenshot 2020-12-29 at 14.40.02.png](https://images.zenhubusercontent.com/5c5c1adbb76da5342d3d6916/8b7e68fa-6664-4b61-9a71-53e82aead76a)

**Comment by vinod-kadam** [comment]
@makkalot  verified on Dev env - issue is no more seen there.

**Comment by ashav-github** [comment]
@vinod-kadam @makkalot, I tested on dev by following the same steps as mentioned in this bug. It worked fine for me. After login to corporate console I am able to see the CCLA sign page and no error thrown. Could complete the CCLA sign-in process successfully.
Tested for 'avdec18foundation | avdec18-clagrp1 | GH user: JyotiM362' 

![zh-2293.jpg](https://images.zenhubusercontent.com/5f6b335ce9044e3973ea11f9/028faf35-85bf-495b-8d01-6c97504c7e26)
